src/grafana/deploy/grafana-configuration/pai-jobview-dashboard.json.template

```diff
@@ -627,14 +627,14 @@
           ]
         },
         "datasource": "PM",
-        "hide": 2,
+        "hide": 0,
         "includeAll": false,
         "label": "",
         "multi": false,
         "multiFormat": "regex values",
         "name": "job",
         "options": [],
-        "query": "label_values(container_label_PAI_JOB_NAME)",
+        "query": "label_values(container_CPUPerc, container_label_PAI_JOB_NAME)",
```

Comment:
use `container_CPUPerc` because if not candidates for the values are from **all** history, but if with `container_CPUPerc`, values are from user defined period of time.